daemon/main.c

```diff
@@ -400,11 +400,24 @@ static const char *verify_required_directory(const char *dir) {
 #ifdef ENABLE_HTTPS
 static void security_init(){
     char filename[FILENAME_MAX + 1];
-    snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
-    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  filename);
+    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  "none");
+    if (!strcmp(security_key, "none")) {
```

Comment:
Why does `"ssl key"` take priority over `"tls key"`?

Reply:
`ssl key` was the start, but software like `MongoDB` are beginning to use `TLS` prefix instead `SSL` prefix, so I arrived in the conclusion that we need to keep the `SSL` prefix to keep compatibility, but it is also a good idea to bring the possibility to use `TLS` prefix, because other projects are doing this.